components/counter/Counter.js

```diff
@@ -14,7 +14,7 @@ class Counter extends PureComponent {
       theme[size],
       {
         [theme[`border-${borderColor}-${borderTint}`]]: borderColor && borderTint,
-        [theme[`border-${borderColor}`]]: !borderTint,
+        [theme[`border-${borderColor}`]]: borderColor && !borderTint,
```

Comment:
I don't think this is the correct solution either? Now you will have a borderColor if borderColor is set and borderTint is not set, but you can have both right? I think the check should just be `borderColor`?

Reply:
Ah no, then you have the combination of borderColor-borderTint. Nevermind.

 :ok: by @InstaK on 15/06/2018 11:02:13